packages/documentation/docs/controls/_content-header_styleguide.md

```diff
@@ -0,0 +1,42 @@
+The content header provides a brief overview of the content on a page. It helps the user to understand what the page is about.
+We typically use it at the very top of the page.
+
+![Content header overview](https://www.figma.com/file/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?type=design&node-id=2250-4784&mode=design&t=XmCepM9jPR9PImPw-4)
+ 
+1. Back button 
+2. Content title
+3. Content subtitle
+4. Action buttons
+
+## Variants
+With our content header variants, you can make it easer for easier for you to achieve a well-balanced visual hierarchy throughout the page.
+
+![Content header variants](https://www.figma.com/file/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?type=design&node-id=2250-9102&mode=design&t=XmCepM9jPR9PImPw-4)
+
+## Options
+- **HasBackButton**: This option can be enabled if you want to provide a way for the user to navigate back.
+- **HeaderTitle**: Here you can set your page title.
+- **HeaderSubtitle**: If you want to provide additional info for your content you can add a descriptive sentence here.
+- **Action buttons**: Enhance the user experience by offering convenient shortcuts for actions that the user might need to perform frequently. Examples are "Add" or "Edit".
+
+## Behavior 
+- **Interaction**: The back button or action buttons can be triggered by pressing anywhere within the button container.
```

Comment:
Not sure what you are trying to say here? Could this be more clear? Is it even necessary to mention this?